addons/ai/functions/fnc_unpackStaticWeaponPFH.sqf

```diff
@@ -0,0 +1,70 @@
+#include "script_component.hpp"
+/*
+ * Author: Ampersand
+ * Move units together if needed and unpacks a static weapon from units' backpacks.
+ *
+ * Arguments:
+ * 0: Gunner <OBJECT>
+ * 1: Assistant <OBJECT>
+ * 2: Target Position <ARRAY>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [_gunner, _assistant] call zen_ai_fnc_unpackStaticWeaponPFH
+ *
+ * Public: No
+ */
+
+params ["_gunner", "_assistant", ["_targetPos", [], [[]], 3]];
+
+if !(local _gunner) exitWith {
+    [QGVAR(unpackStaticWeapon), _this, _gunner] call CBA_fnc_targetEvent;
+};
+
+_gunner setVariable [QGVAR(unpackStaticWeaponTargetPos), _targetPos];
+
+// Close enough, set up weapon
+if (_gunner distance _assistant < 3) exitWith {
```

Comment:
Magic number. Add a define at the top of the function. Also for the other ones, maybe: `DISTANCE_CLOSE`, `MOVE_DELAY`, and `MOVE_TIMEOUT`.

Reply:
https://github.com/zen-mod/ZEN/pull/378/commits/4209bc57c6ff8d0bfdd7b50ccb0a145c61d13349